.github/workflows/check.yml

```diff
@@ -39,35 +39,6 @@ jobs:
           incremental_files_only: false
           config: "cspell.yaml"
 
-  # TODO: migrate this check to nix
-  substrate-check:
-    name: Check Benchmarks
-    runs-on:
-      - self-hosted
-      - linux
-      - x64
-      - sre
-    concurrency:
-      group: check-${{ github.ref }}
-      cancel-in-progress: true
-    container:
-      image: composablefi/ci-linux:2022-08-06
-    steps:
-      - name: Clean up
-        continue-on-error: true
-        run: |
-          sudo chown -R $USER:$USER $GITHUB_WORKSPACE
-          docker system prune --force --all --volumes
-      - uses: actions/checkout@v2
-      - name: Rustup show
-        run: |
-          rustup show
-      - uses: actions/checkout@v2
-      - name: cargo check --all --benches
-        run: |
-          cd code
-          SKIP_WASM_BUILD=1 cargo +nightly check --benches --all --features runtime-benchmarks # checks all rust crates with default-features, tests, benchmarks
```

Comment:
was used all benches with rb feature. are we testing now that with all benches? just to be sure check == run all.

Reply:
I renamed the existing check from `check-benchmarks` to `test-running-of-pallet-benchmarks`.